plugins/search-backend-module-elasticsearch/api-report.md

```diff
@@ -211,10 +224,14 @@ export class ElasticSearchSearchEngine implements SearchEngine {
   }: ElasticSearchOptions): Promise<ElasticSearchSearchEngine>;
   // (undocumented)
   getIndexer(type: string): Promise<ElasticSearchSearchEngineIndexer>;
+  // (undocumented)
+  protected indexTemplate?: ElasticSearchCustomIndexTemplate;
```

Comment:
Any reason to mark this `protected` instead of `private`?

Reply:
I use it in the test https://github.com/backstage/backstage/pull/11962/commits/1b59892019bf4073cbe8251450742f6e33a64d27#diff-a8638b49aa05091d55559838ef44cf5828add0f1aff4965f3e1b0b6baa1a52faR36 and if its private I can't access it.. Do you have any suggestions? If we change to the above implementation this isn't needed though.

Reply:
Ah-hah. Makes sense. Let's go with the put-on-set approach.

Reply:
Updated!